src/assets/scss/layout/layout.scss

```diff
@@ -31,9 +31,11 @@
   }
 }
 
-.footer-styling {
-  padding-bottom: 3em;
-  padding-top: 5em;
+.page-wrapper {
+  position: absolute;
```

Comment:
How come we want to switch the position styling to absolute here? Would this have any effect on the `error` page?

Reply:
also, just to confirm, what is the may styling that is making the `error` page now extend to the entire viewport now?

Reply:
Looking at this again, I could've actually just used `height: 100vh` instead of `position: absolute`. Will make this change because it more directly addresses the issue of the page wrapper not fully extending across the height of the viewport.